samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General config
+CONFIG_FPU=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Rate of cloud interactions
+# These are faster than one normally would use in a low power device.
+# This is strictly for demo purposes.
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS=120
+CONFIG_LOCATION_TRACKING_SAMPLE_INTERVAL_SECONDS=360
+CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES=5
+
+# Logs
+CONFIG_MULTI_SERVICE_LOG_LEVEL_DBG=y
+CONFIG_LOG=y
+CONFIG_NET_LOG=y
+CONFIG_LOG_PRINTK=y
+CONFIG_COAP_LOG_LEVEL_INF=y
+CONFIG_LOCATION_LOG_LEVEL_DBG=y
+CONFIG_DATE_TIME_LOG_LEVEL_DBG=y
+CONFIG_NRF_CLOUD_COAP_LOG_LEVEL_DBG=y
+
+# LTE link control - used by PGPS and main application
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
+#CONFIG_NET_L2_VIRTUAL=y
+
+# Modem
+CONFIG_MODEM_KEY_MGMT=y
+CONFIG_MODEM_JWT=y
+CONFIG_MODEM_INFO_ADD_DEVICE=y
+CONFIG_MODEM_INFO_ADD_DATE_TIME=n
+CONFIG_MODEM_INFO_ADD_SIM=n
+CONFIG_MODEM_INFO_ADD_SIM_ICCID=n
+CONFIG_MODEM_INFO_ADD_SIM_IMSI=n
+
+# Network
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_TLS_SET_MAX_FRAGMENT_LENGTH=y
+CONFIG_NET_SOCKETS_OFFLOAD_PRIORITY=40
+
+# nRF Cloud
+CONFIG_NRF_CLOUD_MQTT=n
+CONFIG_NRF_CLOUD_FOTA=n
+CONFIG_NRF_CLOUD_LOCATION=n
+CONFIG_NRF_CLOUD_COAP=y
+CONFIG_COAP_CLIENT_BLOCK_SIZE=1024
+CONFIG_COAP_CLIENT_STACK_SIZE=6144
+CONFIG_COAP_CLIENT_THREAD_PRIORITY=0
+CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE=40
+CONFIG_NRF_CLOUD_ALERT=y
+CONFIG_NRF_CLOUD_LOG_DIRECT=y
+CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL=3
+CONFIG_NRF_CLOUD_LOG_LOG_LEVEL_INF=y
+CONFIG_NRF_CLOUD_PGPS_REPLACEMENT_THRESHOLD=2
+CONFIG_NRF_CLOUD_PGPS_NUM_PREDICTIONS=4
+
+# Location Services configuration
+CONFIG_LOCATION_DATA_DETAILS=y
+
+# Date Time lib - Used by PGPS and main application
+CONFIG_DATE_TIME_AUTO_UPDATE=y
+CONFIG_DATE_TIME_MODEM=y
+CONFIG_DATE_TIME_NTP=y
+
+CONFIG_NRF_CLOUD_COAP_SERVER_HOSTNAME="coap.nrfcloud.com"
```

Comment:
If this isn't the default value, maybe it should be?

Reply:
It is the default.  I change it to test on other stages, but I can remove it now.